.github/workflows/crucible-scheduled.yaml

```diff
@@ -16,3 +16,9 @@ jobs:
     secrets:
       production_registry_auth: ${{ secrets.CRUCIBLE_PRODUCTION_ENGINES_REGISTRY_AUTH }}
       quay_oauth_token: ${{ secrets.CRUCIBLE_QUAYIO_OAUTH_TOKEN }}
+
+  call-crucible-release:
+    uses: perftool-incubator/crucible/.github/workflows/crucible-release.yaml@main
+    needs: call-core-crucible-ci_scheduled
+    with:
+      github_workspace: "$GITHUB_WORKSPACE"
```

Comment:
You are adding this to a workflow that is echeduled to run monthly, but you have it designed to create quarterly versions.  So that would result in the version being set once (month 1 of the quarter) and then overwritten twice (months 2 & 3 of the quarter).  That seems wrong.  Shouldn't this be only run once per quarter?

Reply:
Yeah, the original idea assumed the job would do nothing if the tag exists.... I am changing for an independent scheduled one.